server/handlers/handlers.go

```diff
@@ -333,10 +333,17 @@ func (ah *ActivityHandler) handleUpdatedActivity(activityIds msteams.ActivityIds
 			return
 		}
 	} else {
-		post, err := ah.plugin.GetAPI().GetPost(postInfo.MattermostID)
-		if err != nil {
-			ah.plugin.GetAPI().LogError("Unable to find the original post", "error", err.Error())
-			return
+		post, postErr := ah.plugin.GetAPI().GetPost(postInfo.MattermostID)
+		if postErr != nil {
+			if strings.Contains(postErr.Error(), "Unable to get the post.") {
+				if err = ah.plugin.GetStore().RecoverPost(postInfo.MattermostID); err != nil {
+					ah.plugin.GetAPI().LogError("Unable to recover the post", "post", post, "error", err)
```

Comment:
```suggestion
					ah.plugin.GetAPI().LogError("Unable to recover the post", "post", post, "error", err.Error())
```

Reply:
There's no fixed convention for this in this plugin.